docs/paper/admin/how-to/basic-troubleshooting.md

```diff
@@ -0,0 +1,135 @@
+---
+slug: /basic-troubleshooting
+---
+
+# Basic Troubleshooting
+
+This guide will help you diagnose your server's problem before reporting it to PaperMC or the plugin's author. Please, read this documentation carefully.
+
+:::tip Backup Your Server
+
+People can break their servers while trying to follow this guide. It is highly advised to backup your server before following this guide.  
+It would be ideal to create a test server by copying your production server's file, but that's not always possible.  
+Please, take a backup first.
+
+:::
+
+:::caution Stop Your Server
```

Comment:
I’m not sure if this is necessary. Stopping the server is somewhat implied by the previous note and things like removing plugins isn’t possible whilst the server is running.

Reply:
I think mentioning both "stop server" and "backup your server" is needed because I've seen few people trying to take a backup while server is alive in `#paper-help` before. This might look dumb, but it is worth mentioning it.